atlassian/parse.go

```diff
@@ -0,0 +1,109 @@
+package atlassian
+
+import (
+	"errors"
+	"strconv"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/jsonquery"
+	"github.com/spyzhov/ajson"
+)
+
+func getTotalSize(node *ajson.Node) (int64, error) {
+	return jsonquery.New(node).ArraySize("issues")
+}
+
+/*
+Records cannot be returned as is from the API. Extra processing is described below.
+ 1. First of all, main properties are located under "fields" key.
+ 2. Secondly, item id is not mirrored under "fields" property, which means
+    unwrapping is not enough, we must attach "id" property.
+
+Visual example of what will happen to each property:
+
+	{
```

Comment:
thanks for example. This is super helpful!

Reply:
:pray: